src/app/visual/phone-field/phone-field-visual.component.html

```diff
@@ -0,0 +1,26 @@
+<div
```

Comment:
I know these are just visual tests, but could we add some headings to each of these to explain the difference between the two? We have a lot of older tests that are difficult to decipher. Since we're starting from scratch here, we have an opportunity to document a bit better.

Reply:
Yup. I actually improved that when I ported these to the demo for the docs. I will port some of that back into these.

Reply:
Done